Jenkinsfile

```diff
@@ -20,6 +20,9 @@ pipeline {
         // R4 V2 endpoints enabled
         AB2D_V2_ENABLED = true
 
+        // Number concurrent benes
+        AB2D_PATIENT_PER_WRITE_THREAD=10
```

Comment:
This variable is not clear to me. Could we rename this to batch size or something? And number concurrent benes isn't really clear.

Reply:
Will change